src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function About() {
   return (
-    <div>About</div>
+    <body>
+      <section className='flex justify-center items-center h-screen w-screen flex-col'>
+        <h1 className=' text-5xl m-10'>About</h1>
+        <img src="/src/images/hero-img.png" className='w-5/6 h 3/6'></img>
+      </section>
+
+      <section className='flex flex-col justify-center items-center'>
+        <article className='flex flex-row w-5/6 m-20'>
+          <div className='flex flex-col align-left '>
```

Comment:
You should set a gap here because the text needs to be more spaced out. I'm also not sure what the purpose of `align-left` is or if it's even a real Tailwind class.